inst/rmarkdown/templates/xaringan/skeleton/skeleton.Rmd

```diff
@@ -14,6 +14,11 @@ output:
 
 ```{r setup, include=FALSE}
 options(htmltools.dir.version = FALSE)
+
+### code output coloring. uncomment to use. 
+### See https://twitter.com/BrodieGaslam/status/1029059002225721344 for a preview
+# old.hooks = fansi::set_knit_hooks(knitr::knit_hooks)
```

Comment:
I guess the new variable `old.hooks` is unlikely to be really useful (those who want colored output probably don't want to restore **knitr** hooks later), so how about not creating this variable?